frontend/src/views/Members.vue

```diff
@@ -411,36 +385,72 @@ export default {
         this.kubeconfigDialog = true
       }
     },
-    confirmDelete (username) {
-      const memberName = escape(displayName(username))
+    async onRemoveUser (name) {
+      const removalConfirmed = await this.confirmRemoveUser(name)
+      if (removalConfirmed) {
+        await this.deleteMember(name)
+        if (this.isCurrentUser(name) && !this.isAdmin) {
+          if (this.projectList.length > 0) {
+            const p1 = this.projectList[0]
+            this.$router.push({ name: 'ShootList', params: { namespace: p1.metadata.namespace } })
+          } else {
+            this.$router.push({ name: 'Home', params: { } })
+          }
+        }
+      }
+    },
+    confirmRemoveUser (name) {
+      const memberName = escape(displayName(name))
       const projectName = escape(this.projectDetails.projectName)
       let messageHtml
-      if (this.isCurrentUser(username)) {
+      if (this.isCurrentUser(name)) {
         messageHtml = `Do you want to remove <span class="red--text text--darken-2 font-weight-bold">yourself</span> from the project <i>${projectName}</i>?`
       } else {
-        messageHtml = `Do you want to delete the member <i>${memberName}</i> from the project <i>${projectName}</i>?`
+        messageHtml = `Do you want to remove the member <i>${memberName}</i> from the project <i>${projectName}</i>?`
```

Comment:
add message slot to ConfirmDialog instead of building html code here which could be vulnerable to xss when using v-html